javascript/javascript1/week1/dogcalculator.js

```diff
@@ -0,0 +1,17 @@
+console.log("Dog Calculator");
+let dogYearOfBirth = 2000;
+let dogYearFuture = 2042;
+let dogYear = dogYearFuture - dogYearOfBirth;
+let dogAge = dogYear * 7;
+
+let shouldShowResultInDogYears = false;
+if (shouldShowResultInDogYears === true) {
+  console.log(
+    "Your dog will be " + dogAge + " years old in " + dogYearFuture + "."
+  );
+} else shouldShowResultInDogYears === false;
+{
+  console.log(
+    "Your dog will be " + dogYear + " human years old in " + dogYearFuture + "."
+  );
+}
```

Comment:
Good work. You have a good structure and some nice logic. 😊

**I have a few observations and suggestions 💫**
- Could the variable dogYear have a different name that represents the value even clearer? 
- Could one of the variables be a constant? 

Tip 💡
In the if else statement you typically define the condition in the if clause - as you have done very nicely. If this condition is false we will automatically move on to the else clause, and here you do not specify a condition. Maybe I can visualize it through pseudocode below. 

if (condition) {
  //  block of code to be executed if the condition is true
} else {
  //  block of code to be executed if the condition is false
}

Here is another source, that might explain it better than me 😇😊
https://www.w3schools.com/js/js_if_else.asp

Reply:
I did think this at the time about if and else statements! And in the next task, I didn't put an else condition, I should've amended - thanks for confirming this!